qa/suites/rados/verify/validater/valgrind.yaml

```diff
@@ -1,6 +1,3 @@
-# see http://tracker.ceph.com/issues/20360 and http://tracker.ceph.com/issues/18126
-os_type: centos
-
```

Comment:
Would be better as:

```
teuthology:
  postmerge:
    - if yaml.os_type ~= 'rocky' then reject() end
```

And perhaps update the comment to reference the PRs Ilya mentioned.

Reply:
The entire `rados/verify` subsuite remains restricted to CentOS Stream by `centos_latest.yaml` symlink one level up, so this PR is basically a no-op that has the side effect of allowing to easily swap that symlink for e.g. Rocky.  That seems to be the intent -- it's just the title of the PR that is misleading.

The change that you are proposing would do something else.

Reply:
> But the larger issue is that the valgrind yaml still can only be used, at this time, with an rpm distro.

Here is my stab at changing that: https://github.com/ceph/ceph/pull/66205.  I only changed the RBD suite there, but if there is consensus on the suppression I can tack on a similar change for the RADOS suite.